<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{
background-color: rgb(61, 61, 61);}</string>
  </property>
  <widget class="QPushButton" name="open_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Apri</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="dipendenti_layout"/>
  </widget>
  <widget class="QPushButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Nuovo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
